clients/src/main/java/org/apache/kafka/clients/consumer/internals/PartitionAssignor.java

```diff
@@ -43,6 +37,7 @@
  * userData in the returned Subscription. For example, to have a rack-aware assignor, an implementation
  * can use this user data to forward the rackId belonging to each member.
  */
+@Deprecated
```

Comment:
The existing PartitionAssignor is deprecated, and reverted back to its 2.3 state since all changes made since are migrated to the new interface

Reply:
Nice work!